index.css

```diff
@@ -0,0 +1,35 @@
+html{
+    font-size: 62.5%;
+}
+body{
+    background-color: var(--half-white);
+   
+}
+.index-container{
+    height: 80vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    
+}
+ol{
+    background: var(--white);
+    border: 1px solid var(--purple-700);
+    padding: 3rem;
+    width: 20rem;
+}
+ol li{
+    margin: 5px 0;
```

Comment:
you can add hover effect list items so the user will know where exactly he is interacting.

Reply:
sure , thank you for the suggestion